client/src/widget/EmbeddedScheduler.js

```diff
@@ -0,0 +1,184 @@
+// Esssentially a copy of the `Scheduler.js` component
+const domain = "http://localhost:3000";
```

Comment:
You should move this down to line 20 to avoid `import in body of module` errors, also you may be able to set this to `window.location.origin` to make your life easier when deploying (that also may not work so let me know if I'm wrong here)

Reply:
Turns out this value needs to be set at build time to send request to the proper location. I think I will just stick to localhost for now lol